articles/cms/wordpress/configuration.md

```diff
@@ -25,7 +25,11 @@ You'll need to be logged into your Auth0 account before starting the steps below
 
 First, we'll check for the Client created for your WordPress site. 
 
-1. Navigate to the [Clients](${manage_url}/#/clients) page and look for a client that is similar to your site name; if you don't find one, it means that a Client was not created by the Wizard. Restart the Setup Wizard and try again.  
+1. Navigate to the [Clients](${manage_url}/#/clients) page and look for a Client that is similar to your site name; if you don't find one, it means that a Client was not created by the Wizard. Restart the Setup Wizard or create a new Client:
+    
+    1. Click **Create Client**
```

Comment:
Maybe change this to a bulleted list? Looks kind of funny having step 1 under step 1

![image](https://user-images.githubusercontent.com/11715799/37724490-24db88d6-2cee-11e8-895d-483c80f4374f.png)

Reply:
Bullet points are for non-sequential items, where order doesn't matter. I could combine into the paragraph separated by commas?

Reply:
yeap, sounds good

Reply:
@mpaktiti - OK, changed!